tests/subsys/suit/fetch/boards/native_sim.overlay

```diff
@@ -7,12 +7,30 @@
 &flash0 {
 	partitions {
 		compatible = "fixed-partitions";
-		#address-cells = <1>;
-		#size-cells = <1>;
+		#address-cells = < 0x1 >;
+		#size-cells = < 0x1 >;
```

Comment:
undo change, address and size cells should be decimal not hex, and without spaces

Reply:
fixed